### ISSUE TITLE
Architect describes app "phases"

### ISSUE THREAD

**Issue by avidrucker** [avidrucker/autofocus-exp#48]
which correlate to primary user stories such as:
```
- user is welcomed to app
- user is prompted to enter in todo items
- user is shown the entire list of todo items saved in app
- user is prompted to choose what they want to do most by dotting items
- user is shown a screen/message that acknowledges they are working on todo item X
- user is prompted to "close out tickets" ( mark a todo item as completed ) as well as whether they want to "open a new duplicate ticket" ( create a duplicate todo item) to continue further work
```

- [ ] Architect converts the above to user stories
- [ ] Architect adds app_phases.md with the above user stories
- [ ] Architect links to app_phases.md under supplemental docs heading in readme

**Comment by avidrucker** [comment]
At a broad level, there are 3 modes:
1. create todos
1. review todos
1. focus on completing one todo

Each mode can be further broken down into sub-modes. There is also an "admin" mode which is really just the user saying they are ready for a new blank list, or to archive (hide) or delete all (or just the completed) items. Once page view has been implemented, admin mode can be replaced with toggle buttons, which will selectively show/hide completed items, archived items, marked items, etc..

**Comment by avidrucker** [comment]
Marking closed as this issue is effectively addressed for now with the resolution of issue #47 via pull request issue #342
